public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -907,6 +951,10 @@ public SurveyQuestionResult apply(AchillesResult o) {
 
         List<AchillesAnalysis> analyses = achillesAnalysisDao.findSubSurveyAnalysisResults(surveyConceptId, questionConceptIds, new String("%"+resultConceptId+"%"));
 
+        if (analyses == null) {
```

Comment:
Can never be null since 
`achillesAnalysisDao.findSubSurveyAnalysisResults(surveyConceptId, questionConceptIds, new String("%"+resultConceptId+"%"))` will return an empty list when no results are found.